docs/en/Overview.md

```diff
@@ -8,13 +8,17 @@ Welcome to Alluxio Documentation! You will find resources regarding deploying Al
 ## Get Started with Alluxio
 
 ### Install Alluxio Locally 
-Check out our [Quick Start Guide]({{ '/en/Get-Started.html' | relativize_url }}) for step-by-step guidance on how to download and configure Alluxio, run Alluxio on a local machine, and perform basic tasks via Alluxio Shell.
+Check out our [Quick Start Guide]({{ '/en/deploy/Get-Started.html' | relativize_url }}) for step-by-step guidance on how to download and configure Alluxio, run Alluxio on a local machine, and perform basic tasks via Alluxio Shell.
```

Comment:
remove

Reply:
i moved the file back to the deploy folder